src/routes/Root/Root.tsx

```diff
@@ -29,14 +84,16 @@ const Root = () => {
           }}
         />
       </header>
-      <main className="p-3">
+      <main>
         <Routes>
           <Route index element={<Home />} />
           <Route path={`${t('slugs.profile.index')}/*`} element={<Profile />} />
-          <Route path={`${t('slugs.about-us')}/*`} element={<AboutUs />} />
+          <Route path={`${t('slugs.about-us')}/*`} element={<Instructions />} />
```

Comment:
Getting `No match` for the [About us and instructions](http://localhost:5173/en/instructions/about-us) link